opengrok-indexer/src/main/java/org/opengrok/indexer/configuration/GraphiteConfig.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * CDDL HEADER START
+ *
+ * The contents of this file are subject to the terms of the
+ * Common Development and Distribution License (the "License").
+ * You may not use this file except in compliance with the License.
+ *
+ * See LICENSE.txt included in this distribution for the specific
+ * language governing permissions and limitations under the License.
+ *
+ * When distributing Covered Code, include this CDDL HEADER in each
+ * file and include the License file at LICENSE.txt.
+ * If applicable, add the following below this CDDL HEADER, with the
+ * fields enclosed by brackets "[]" replaced with your own identifying
+ * information: Portions Copyright [yyyy] [name of copyright owner]
+ *
+ * CDDL HEADER END
+ */
+
+/*
+ * Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+ */
+
+package org.opengrok.indexer.configuration;
+
+import io.micrometer.graphite.GraphiteProtocol;
+
+public class GraphiteConfig {
+    private int port;
+    private String host;
+    private boolean enabled;
+    private GraphiteProtocol protocol;
```

Comment:
Can this have a default value?

Reply:
sure. which one do you think is reasonable ?

Reply:
`isEnabled()` does not check the `protocol` field so it will remain set to `null`. The `GraphiteConfig#setProtocol()` will choose `GraphiteProtocol.PICKLED` in such case.